### DISCUSSION TITLE
2022-11-17 Meeting Agenda

### DISCUSSION THREAD

**Discussion by aflackey** [ucum-org/ucum#254] [Meetings]
### Date: 2022-11-16, 3:00 – 4:00 pm EST
**Chair(s): Marjorie Rallins**

|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
| ----------- | -----| -----|
| 3:00 | Welcome | Marjorie Rallins |
| 3:00 - 3:05 | Provisional member | Marjorie Rallins |
| 3:05 - 3:15 | Cooperation with CIPM on Digital-SI | Simon Cox |
| 3:15 - 3:20 | New releases of UCUM | Eza Hafeza |
| 3:20 - 3:50 | Continue unit discussions | Eza Hafeza and others |
||•	[Newtonian constant of gravitation](https://github.com/ucum-org/ucum/issues/233) ||
||•	[Proton mass](https://github.com/ucum-org/ucum/issues/232)||
||•	[Electron mass](https://github.com/ucum-org/ucum/issues/231)||
||•	[Permittivity of vacuum](https://github.com/ucum-org/ucum/issues/230)||
||•	[Viable cell density](https://github.com/ucum-org/ucum/issues/224)||
||•	[Parts per quadrillion](https://github.com/ucum-org/ucum/issues/223)||
||•	[Platinum cobalt unit](https://github.com/ucum-org/ucum/issues/222)||
| 3:50 - 3:55 | 2023 meeting planning | April Lackey |
|3:55 - 4:00 | Status of GitHub migration | Tim Briscoe |
|4:00|Adjourn|Eza Hafeza|

***OPEN ACTION ITEMS***
|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
| ----------- | -----| -----| -----|
|Add Josef Aerts to UCUM’s GitHub|10/3/2022| 10/7/2022| Tim Briscoe|
|Complete remaining Trac>GitHub migration|10/3/2022| 10/7/2022| Tim Briscoe|
|[kIU/l to KUA/l](https://github.com/ucum-org/ucum/issues/180) – Propose k[UA]/l to requestor, advising current use of U in UCUM standard; invite requestor to follow and/or engage in the GitHub issue|10/3/2022||Eza Hafeza|
|[Chemiluminescence](https://github.com/ucum-org/ucum/issues/182) – Request more info from the submitter: context, relativeness to base units|10/3/2022||Eza Hafeza|
|[Turbidity (multiple unit requests)](https://github.com/ucum-org/ucum/issues?q=is%3Aissue+is%3Aopen+turbidity) – Request more information from submitter: context, relativeness to base units|10/3/2022||Eza Hafeza|
|Develop formal [criteria for new unit request](https://github.com/ucum-org/ucum/issues/200)|10/3/2022||Eza Hafeza|

***LINKS TO REFERENCE MATERIALS***
[Cooperation with CIPM on Digital-SI.ProvBy Simon COx](https://drive.google.com/file/d/159QJncutSKxj3FT06uxTzoAcRC5tKvUi/view?usp=share_link)
[2022-10-03 Meeting Summary](https://docs.google.com/document/d/16JdAOu0Vux7OQAs6raG3xgRBk4ekNbOK/edit?usp=share_link&ouid=110875911742539584915&rtpof=true&sd=true)
[All open issues on UCUM GitHub](https://github.com/ucum-org/ucum/issues/200)

@ucum-org/regenstrief 
@ucum-org/advisors

**Comment by chgessner** [comment]
confusing: eMail subject points tomorrow. But meeting is apparently scheduled today!

On 16 November 2022 17:15:21 CET, aflackey ***@***.***> wrote:
>### Date: 2022-11-16, 3:00 – 4:00 pm EST
>
>**Chair(s): Marjorie Rallins**
>
>
>
>|**TIME** |**AGENDA TOPICS** |**LEAD(S)** |
>
>| ----------- | -----| -----|
>
>| 3:00 | Welcome | Marjorie Rallins |
>
>| 3:00 - 3:05 | Provisional member | Marjorie Rallins |
>
>| 3:05 - 3:15 | Cooperation with CIPM on Digital-SI | Simon Cox |
>
>| 3:15 - 3:20 | New releases of UCUM | Eza Hafeza |
>
>| 3:20 - 3:50 | Continue unit discussions | Eza Hafeza and others |
>
>||•	[Newtonian constant of gravitation](https://github.com/ucum-org/ucum/issues/233) ||
>
>||•	[Proton mass](https://github.com/ucum-org/ucum/issues/232)||
>
>||•	[Electron mass](https://github.com/ucum-org/ucum/issues/231)||
>
>||•	[Permittivity of vacuum](https://github.com/ucum-org/ucum/issues/230)||
>
>||•	[Viable cell density](https://github.com/ucum-org/ucum/issues/224)||
>
>||•	[Parts per quadrillion](https://github.com/ucum-org/ucum/issues/223)||
>
>||•	[Platinum cobalt unit](https://github.com/ucum-org/ucum/issues/222)||
>
>| 3:50 - 3:55 | 2023 meeting planning | April Lackey |
>
>|3:55 - 4:00 | Status of GitHub migration | Tim Briscoe |
>
>|4:00|Adjourn|Eza Hafeza|
>
>
>
>***OPEN ACTION ITEMS***
>
>|**ITEM** |**DATE IDENTIFIED**|**TARGET COMPLETION**| **LEAD** |
>
>| ----------- | -----| -----| -----|
>
>|Add Josef Aerts to UCUM’s GitHub|10/3/2022| 10/7/2022| Tim Briscoe|
>
>|Complete remaining Trac>GitHub migration|10/3/2022| 10/7/2022| Tim Briscoe|
>
>|[kIU/l to KUA/l](https://github.com/ucum-org/ucum/issues/180) – Propose k[UA]/l to requestor, advising current use of U in UCUM standard; invite requestor to follow and/or engage in the GitHub issue|10/3/2022||Eza Hafeza|
>
>|[Chemiluminescence](https://github.com/ucum-org/ucum/issues/182) – Request more info from the submitter: context, relativeness to base units|10/3/2022||Eza Hafeza|
>
>|[Turbidity (multiple unit requests)](https://github.com/ucum-org/ucum/issues?q=is%3Aissue+is%3Aopen+turbidity) – Request more information from submitter: context, relativeness to base units|10/3/2022||Eza Hafeza|
>
>|Develop formal [criteria for new unit request](https://github.com/ucum-org/ucum/issues/200)|10/3/2022||Eza Hafeza|
>
>
>
>***LINKS TO REFERENCE MATERIALS***
>
>[Cooperation with CIPM on Digital-SI.ProvBy Simon COx](https://drive.google.com/file/d/159QJncutSKxj3FT06uxTzoAcRC5tKvUi/view?usp=share_link)
>
>[2022-10-03 Meeting Summary](https://docs.google.com/document/d/16JdAOu0Vux7OQAs6raG3xgRBk4ekNbOK/edit?usp=share_link&ouid=110875911742539584915&rtpof=true&sd=true)
>
>[All open issues on UCUM GitHub](https://github.com/ucum-org/ucum/issues/200)
>
>
>
>-- >
>Reply to this email directly or view it on GitHub:
>
>https://github.com/orgs/ucum-org/discussions/254
>
>You are receiving this because you are subscribed to this thread.
>
>
>
>Message ID: ***@***.***>
-- 
Sent from a mobile device. Please excuse my brevity.

**Reply by aflackey** [reply]
Yes, my apologies. Meeting is today, 16th November.

**Comment by aflackey** [comment]
Meeting summary posted here: [UCUMCommittee_MeetingSummary_2022-11-16.docx](https://github.com/ucum-org/ucum/files/10060789/UCUMCommittee_MeetingSummary_2022-11-16.docx)

@ucum-org/advisors 
@ucum-org/regenstrief
